pywebostv/controls.py

```diff
@@ -298,6 +298,69 @@ class InputControl(WebOSControlBase):
         "info": {
             "command": [["type", "button"], ["name", "INFO"]]
         },
+        "one": {
```

Comment:
Would you perhaps consider renaming "one" to "num_1" (and all the digits below) ?

Reasoning: If I had to send the sequence: "123", I'd probably do something like:

```
for key in sequence:
     getattr(control, "num_" + key)(..)
```

With the digits spelled out, I'd have to maintain an lookup dictionary.

What do you think?

Reply:
Yeah, no problem. Your repo. Your style.